libs/shared/utils/src/getPaths/isSlugRestricted.ts

```diff
@@ -1,7 +1,7 @@
 import { restrictedSlugs } from './restrictedSlugs';
 
 export const isSlugRestricted = (slug: string): boolean => {
-  return !restrictedSlugs.some((restrictedSlug) =>
-    slug.includes(restrictedSlug),
+  return restrictedSlugs.some((restrictedSlug) =>
+    new RegExp(restrictedSlug, 'i').test(slug),
```

Comment:
Is there any particular functional advantage to the switch to a regex here?

For now, the `restrictedSlugs` are all plain strings. I'm guessing this is in anticipation of wanting to tighten the matching on `restrictedSlugs` in the future (or even the present)?

Reply:
Oooh, now I see the odd semantics that were here before... I agree, it makes much more sense for the `.some()` ***not*** to be negated within `isSlugRestricted()` here but instead to have the negation at the point of use in `getPaths.ts`.

Reply:
You're right. We don't need the RegExp for case insensitivity. I will change it.

Reply:
Also what I wrote is just plain wrong, i.e., won't fix the test-page issue. 😅